tests/console/zypper_docker.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
```

Comment:
Please use only the years during which you worked on this file, e.g. `Copyright © 2018-2019 SUSE LLC`